File: Server/Main/Source/StampieAppServer/Data/Entities/Stamp.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

using StampieAppServer.Data.Codebooks;

namespace StampieAppServer.Data.Entities
{
    public class Stamp : IEntity
    {
        [Key]
        public Guid Id { get; set; }

        [Required]
        public StampType Type { get; set; }

        [Required]
        public string StampId { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Category { get; set; }

        [Required]
        public string County { get; set; }

        [Required]
        public DateTime Published { get; set; }

        public string SellingPlace1 { get; set; }
        public string SellingPlace1Web { get; set; }
        public string SellingPlace2 { get; set; }
        public string SellingPlace2Web { get; set; }
        public string SellingPlace3 { get; set; }
        public string SellingPlace3Web { get; set; }
        public string SellingPlace4 { get; set; }
        public string SellingPlace4Web { get; set; }
        public string SellingPlace5 { get; set; }
        public string SellingPlace5Web { get; set; }
        public string SellingPlace6 { get; set; }
        public string SellingPlace6Web { get; set; }
        public string SellingPlace7 { get; set; }
        public string SellingPlace7Web { get; set; }
        public string SellingPlace8 { get; set; }
        public string SellingPlace8Web { get; set; }
        public string SellingPlace9 { get; set; }
        public string SellingPlace9Web { get; set; }
        public string SellingPlace10 { get; set; }
        public string SellingPlace10Web { get; set; }
        public string SellingPlace11 { get; set; }
        public string SellingPlace11Web { get; set; }

        public double GpsPositionLat { get; set; }
        public double GpsPositionLng { get; set; }
    }
}

File: Android/app/src/main/java/cz/kalina/stampie/data/entities/Stamp.java
package cz.kalina.stampie.data.entities;

import java.util.Date;

import cz.kalina.stampie.utils.StampRecord;

public class Stamp extends BaseEntity {

    private String      name;
    private String      category;
    private String      county;
    private Date        published;
    private String      sellingPlace1;
    private String      sellingPlace1Web;
    private String      sellingPlace2;
    private String      sellingPlace2Web;
    private String      sellingPlace3;
    private String      sellingPlace3Web;
    private String      sellingPlace4;
    private String      sellingPlace4Web;
    private String      sellingPlace5;
    private String      sellingPlace5Web;
    private String      sellingPlace6;
    private String      sellingPlace6Web;
    private String      sellingPlace7;
    private String      sellingPlace7Web;
    private String      sellingPlace8;
    private String      sellingPlace8Web;
    private String      sellingPlace9;
    private String      sellingPlace9Web;
    private String      sellingPlace10;
    private String      sellingPlace10Web;
    private String      sellingPlace11;
    private String      sellingPlace11Web;
    private Double      gpsPositionLat;
    private Double      gpsPositionLng;

    public Stamp() {
    }

    public Stamp(String name, String category, String county, Date published, Double gpsPositionLat, Double gpsPositionLng) {
        this.name = name;
        this.category = category;
        this.county = county;
        this.published = published;
        this.gpsPositionLat = gpsPositionLat;
        this.gpsPositionLng = gpsPositionLng;
    }

    public Stamp(StampRecord stampRecord) {
        this.name = stampRecord.name;
        this.category = stampRecord.category;
        this.county = stampRecord.county;
        this.published = stampRecord.published;
        this.gpsPositionLat = stampRecord.gpsPositionLat;
        this.gpsPositionLng = stampRecord.gpsPositionLng;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getCounty() {
        return county;
    }

    public void setCounty(String county) {
        this.county = county;
    }

    public Date getPublished() {
        return published;
    }

    public void setPublished(Date published) {
        this.published = published;
    }

    public double getGpsPositionLat() {
        return gpsPositionLat;
    }

    public void setGpsPositionLat(double gpsPositionLat) {
        this.gpsPositionLat = gpsPositionLat;
    }

    public double getGpsPositionLng() {
        return gpsPositionLng;
    }

    public void setGpsPositionLng(double gpsPositionLng) {
        this.gpsPositionLng = gpsPositionLng;
    }

    @Override
    public String toString() {
        return "Stamp [id=" + id + ", name=" + name + ", category=" + category + ", county=" + county + ", published=" + published.toString() +
                ", gpsPositionLat=" + gpsPositionLat + ", gpsPositionLng=" + gpsPositionLng + "]";
    }
}
